ministryofjustice/parliamentary-questions
# CDPT-530 PQ - 1.7 - Replace accordions with select dropdown

## Description
Replacing each accordion with a drop down may work for the ‘Filter' section but not the 'Quick actions’ section, where the functionality cannot be represented as one choice out of 'x' options.  Instead I have added aria-expanded attributes to the trigger elements. These have a boolean value which are toggled and should declare if the forms are visible or not.

## Self-review checklist


### Screenshots

### Quick Actions section:

![quick-actions-section](https://github.com/ministryofjustice/parliamentary-questions/assets/6988369/680ed2a3-d377-4529-a740-03ecade11e62)

### Code inspection of the 'Edit PQ Dates' form:

<img width="835" alt="edit-pq-dates-inspection" src="https://github.com/ministryofjustice/parliamentary-questions/assets/6988369/a993e0ff-9e5d-4777-9730-00c31ab96f07">

### Related JIRA tickets

https://dsdmoj.atlassian.net/jira/software/c/projects/CDPT/boards/864?assignee=557058%3A2d2f4a33-f427-483f-9c36-338caee60eae&selectedIssue=CDPT-530

### Deployment

N/A
### Manual testing instructions

Please use your browser Developer Tools to inspect the collapsed 'Quick actions' and 'Filter' options in both the 'New' and 'In progress' pages.  The expand/collapse buttons as they are toggled should have their associated aria-expanded attribute toggle true/false.  This should be tested along with the aria-expanded toggling when clicking the 'Cancel' buttons in the 'Quick Actions' section forms.
